proto/data.proto

```diff
@@ -148,8 +148,14 @@ message UpdateMutation {
   map<uint32, Actors> actors = 1;
 }
 
+message RowIdStepInfo {
```

Comment:
How about `RowIdGenRule`?

Reply:
Cool, that sounds better. Will rename it.